data/frameSimulator/src/PercivalFrameSimulatorPlugin.cpp

```diff
@@ -0,0 +1,220 @@
+
+
+#include "version.h"
+#include "../include/PercivalFrameSimulatorPlugin.h"
```

Comment:
Are you sure this needs to be a relative include path here?  What happens when these files get installed?

Reply:
fixed.